### PR TITLE
Support background color for Compose

### PR DESCRIPTION
**Pull request by jalalkun**
Tracked by [ABCD-XXXX](https://github.com/warting/android-signaturepad/issues/ABCD-XXXX)

## This PR Support background colour for Compose

_My project needs to change the background colour in Jetpack Compose._
I'd add a parameter background at [SignaturePadView](signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt)

## Considerations and implementation

_What technical details should the team pay particular attention to? What unexpected issues did you encounter?_

### How to test

1. _How can others test your changes?_

### Test(s) added 

_Why did you add tests around the areas you did? If none, explain why_

### Screenshots

| Before | After |
| ------ | ----- |
| _Screenshot_20221222-172311_Signaturepad_ | _Screenshot_20221222-172712_Signaturepad_ |

### CODE REVIEW

**Comment by warting** [comment]
@jalalkun Thank you for your contribution. What do you think of exposing the modifier as a optional parameter as suggester here instead?
https://twitter.github.io/compose-rules/rules/#when-should-i-expose-modifier-parameters
https://chris.banes.me/posts/always-provide-a-modifier/

**Comment by jalalkun** [comment]
> @jalalkun Thank you for your contribution. What do you think of exposing the modifier as a optional parameter as suggester here instead? https://twitter.github.io/compose-rules/rules/#when-should-i-expose-modifier-parameters https://chris.banes.me/posts/always-provide-a-modifier/

I think your suggestion is better, I'll try

**Comment by jalalkun** [comment]
please review again @warting

File: signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt [hunk 1]
```diff
@@ -17,6 +18,7 @@ import se.warting.signatureview.views.SignedListener
 @SuppressWarnings("LongParameterList")
 @Composable
 fun SignaturePadView(
+    modifier: Modifier = Modifier,
```
Comment (warting) [inline signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt:21]:
```suggestion
    modifier: Modifier = Modifier.fillMaxSize(),
```

Comment (warting) [inline signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt:21]: This changes the default behavior and is no longer fillMaxSize(). But that might be fine?

Comment (warting) [reply signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt:21]: And with this in place we can update the sample and get rid of the surrounding Box :+1: 

SignaturePadView(
      modifier = Modifier
          .height(SIGNATURE_PAD_HEIGHT.dp)
          .fillMaxWidth()
          .border(
              width = 2.dp,
              color = Color.Red,
          )

File: signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package se.warting.signaturepad
 
 import android.graphics.Bitmap
+import androidx.compose.foundation.background
```
Comment (warting) [inline signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt:4]:
```suggestion
```

File: signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt (hunk 1 above)
Comment (jalalkun) [reply signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt:21]: I think fine, we didn't need a box anymore to set the size.
I have tried with 
`SignaturePadView(modifier = Modifier.background(Color.Green))`
inside the box, SignaturePadView size is following the box size. 
without a box, the whole screen will show the SignaturePadView like using `fillMaxSize()` and `fillMaxHeight()` in the same time

**Comment by warting** [comment]
@jalalkun run :signature-pad:apiDump and then we are good to merge!

**Comment by jalalkun** [comment]
> @jalalkun run :signature-pad:apiDump and then we are good to merge!

done
